app/components/dashboard/collection_component.html.erb

```diff
@@ -5,6 +5,12 @@
         <span class="fas fa-pencil-alt"></span>
       <% end %>
     <% end %>
+
+    <% if allowed_to?(:delete?, collection) %>
+      <%= link_to collection_path(collection), method: :delete, id: "remove-collection-#{collection.id}", aria: { label: "Delete #{name}"}, data: {confirm: "Are you sure you want to delete this draft collection? It cannot be undone."} do %>
```

Comment:
why does this need an id? Would you mind adding spaces around the brackets in the hash?

Reply:
fixed spacing, removed id in component as i got the aria-label finder to work in the test